### PR TITLE
External display: ignore play in background setting(closes #269)

### PR DESCRIPTION
**Pull request by Mikanbu**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md) and run `bundle exec fastlane lint` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### CODE REVIEW

File: Sources/VLCPlaybackController.m [hunk 1]
```diff
@@ -1237,15 +1237,20 @@ - (void)scheduleSleepTimerWithInterval:(NSTimeInterval)timeInterval
     _sleepTimer = [NSTimer scheduledTimerWithTimeInterval:timeInterval target:self selector:@selector(stopPlayback) userInfo:nil repeats:NO];
 }
 
+- (BOOL)isPlayingOnExternalScreen
+{
+    return (_renderer || [[UIDevice currentDevice] VLCHasExternalDisplay]);
```
Comment (carolanitz) [inline Sources/VLCPlaybackController.m]: I think you don't need the brackets here ;)

Comment (Mikanbu) [reply Sources/VLCPlaybackController.m]: Indeed but I found it nicer to have them :D

**Comment by carolanitz** [comment]
can you adjust the testcases to add this case

**Comment by carolanitz** [comment]
merged with f8165139d03aa74b68d237c0bcc572f7bfae0b94

**Pull request closed without merging**